### PR TITLE
Lead by example: lint PORO model

### PR DESCRIPTION
**Pull request by bf4**

### CODE REVIEW

**Comment by bf4** [comment]
@joaomdmoura This, arguably, was missing from https://github.com/rails-api/active_model_serializers/pull/996

File: test/fixtures/poro.rb [hunk 1]
```diff
@@ -13,12 +13,8 @@ def cache_key
...
   end
 
-  def cache_key_with_digest
-    "#{cache_key}/#{FILE_DIGEST}"
-  end
-
-  def updated_at
-    @attributes[:updated_at] ||= DateTime.now.to_time
+  def serializable_hash(options = nil)
+    @attributes
```
Comment (bf4) [inline test/fixtures/poro.rb]: Interestingly, this method was missing

File: test/fixtures/poro.rb [hunk 2]
```diff
@@ -42,6 +41,14 @@ def method_missing(meth, *args)
       super
     end
   end
+
+  def cache_key_with_digest
+    "#{cache_key}/#{FILE_DIGEST}"
+  end
+
+  def updated_at
+    @attributes[:updated_at] ||= DateTime.now.to_time
```
Comment (bf4) [inline test/fixtures/poro.rb]: This should really be `Time.now`.  We can be pretty confident we're using the Gregorian calendar, no?

**Comment by joaomdmoura** [comment]
:+1:

File: test/fixtures/poro.rb (hunk 1 above)
Comment (joaomdmoura) [inline test/fixtures/poro.rb]: :+1:

File: test/fixtures/poro.rb (hunk 2 above)
Comment (joaomdmoura) [inline test/fixtures/poro.rb]: I don't mind keeping it :smile:

Comment (bf4) [inline test/fixtures/poro.rb]: I didn't change it in this PR. :) OT: In 1.8 and 1.9 I think DateTime was much slower than Time.  Now, I think avoiding DateTime is mostly about it being the wrong tool and being weird with timezones sometimes...

**Pull request merged**